crates/evm/src/executor/inspector/cheatcodes/fs.rs

```diff
@@ -13,19 +14,19 @@ use walkdir::WalkDir;
 
 fn project_root(state: &Cheatcodes) -> Result {
     let root = state.config.root.display().to_string();
-    Ok(abi::encode(&[Token::String(root)]).into())
+    Ok(DynSolValue::String(root).abi_encode().into())
 }
 
 fn read_file(state: &Cheatcodes, path: impl AsRef<Path>) -> Result {
     let path = state.config.ensure_path_allowed(path, FsAccessKind::Read)?;
     let data = fs::read_to_string(path)?;
-    Ok(abi::encode(&[Token::String(data)]).into())
+    Ok(DynSolValue::String(data).abi_encode().into())
```

Comment:
Yea IMO we should be able to do `data.abi_encode()` w/o having to wrap the type ourselves

Reply:
https://github.com/foundry-rs/foundry/pull/5928#discussion_r1346568383